examples/docker-mixin-test/README.md

```diff
@@ -1,43 +1,30 @@
-# My First Bundle
+# Docker Mixin Example Bundle
 
-This is an empty bundle that porter has created to get you started!
+This is an example bundle to show how to use the Docker mixin in Porter!
 
 # Contents
 
 ## porter.yaml
 
 This is the porter manifest. See https://porter.sh/author-bundles/ for 
 details on every field and how to configure your bundle. This is a required
-file.
+file. 
 
-## helpers.sh
+The commands available in the docker mixin are docker pull, push, build, 
+run, remove, and login. 
 
-This is a bash script where you can place helper functions that you can call
-from your porter.yaml file.
+### Notes on docker login
+Login has two optional parameters - username and password. To pass in the 
+username and password, you can either enter them in the yaml or set them 
+as environment variables and pass them in as credentials. In order to pass 
+them in as credentials, you need to add the credentials section in the yaml. 
+You need to run porter credentials generate and specify where the values come
```

Comment:
At the moment, it's kind of hard to find some of these commands and know what to do with them so maybe we should link to them.

```suggestion
You need to run [porter credentials generate](https://porter.sh/cli/porter_credentials_generate/) and specify where the values come
```